ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/AmountRangeMapper.java

```diff
@@ -7,14 +7,23 @@
 @Component
 public class AmountRangeMapper implements BuilderMapper<CCDCase, AmountRange, CCDCase.CCDCaseBuilder> {
 
+    private final MoneyMapper moneyMapper;
+
+    public AmountRangeMapper(MoneyMapper moneyMapper) {
+        this.moneyMapper = moneyMapper;
+    }
+
     @Override
     public void to(AmountRange amountRange, CCDCase.CCDCaseBuilder builder) {
-        amountRange.getLowerValue().ifPresent(builder::amountLowerValue);
-        builder.amountHigherValue(amountRange.getHigherValue());
+        builder.amountLowerValue(amountRange.getLowerValue().map(moneyMapper::to).orElse(null));
+        builder.amountHigherValue(moneyMapper.to(amountRange.getHigherValue()));
     }
 
     @Override
     public AmountRange from(CCDCase ccdAmountRange) {
-        return new AmountRange(ccdAmountRange.getAmountLowerValue(), ccdAmountRange.getAmountHigherValue());
+        return AmountRange.builder()
```

Comment:
Good work. Can you pls remove the constructor in AmountRange. If we have mix of builder and constructor, it will never get cleaned.

Reply:
We need that constructor as well, We have our @Builder annotation on those constructors.

Reply:
The Builder is not anotated on constructor. Pls check

Reply:
It is now :)